packages/api-server/api_server/routes/tasks/tasks.py

```diff
@@ -138,6 +139,74 @@ async def get_task_state(
     return result
 
 
+@router.get("/queue_entry", response_model=List[TaskQueueEntryPydantic])
+async def query_task_queue_entry(
```

Comment:
Add docs explaining what it does, how is it different from querying task state.

Reply:
https://github.com/open-rmf/rmf-web/pull/887/commits/14261193e7566e734d847a18d78e9b11f7680d49